hs/src/PtrlConsts.hs

```diff
@@ -0,0 +1,14 @@
+module PtrlConsts
+    (
+     PtrlConsts(..)
+    ) where
+
+import Coin (Coin(..))
+
+data PtrlConsts =
+  PtrlConsts
+  { -- |The linear factor for the minimum fee calculation
+    minfeeA :: Coin
+    -- |The constant factor for the minimum fee calculation
+  , minfeeB :: Coin
```

Comment:
Why are these of type `Coin` ?

Reply:
is it odd? `minfeeA` would be the number of lovelace per byte that you must pay as a fee, and `minfeeB` is an additional constant number of lovelace as a constant factor. would it be more natural to convert to `Coin` only at the end?

Reply:
hm, I had this in mind as general parameters, e.g., parameters for a linear function like `y=a*x + b` where type of `Coin` for `a` would not make much sense, right?

Reply:
I'm happy either way. In general, I try not to pull out the underlying nat inside the coin, so thinking of the coin as a value. Then the type of `a` and `b` would have to be `Coin`. And it makes sense to me that `a` is the number of coins per byte. But maybe that's just me, I don't know. :)